.vscode/settings.json

```diff
@@ -15,8 +15,7 @@
     "Dockerfile.*": "dockerfile",
```

Comment:
The PR title uses the `fix` prefix, but I think it should be `chore`? The current `fix` prefix would make the release action bump the patch version of Renovate.